src/app.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
 import Settings from './components/settings/Settings';
 import SearchBar from './components/searchbar/SearchBar';
+import Button from './components/button/Button';
```

Comment:
I would rename this to something else. Button is a bit too generic. SearchButton is a bit better but if u can think of something else, go with that